js/live-editor.js

```diff
@@ -195,9 +177,11 @@ window.LiveEditor = Backbone.View.extend({
         this.handleMessagesBound = this.handleMessages.bind(this);
         $(window).on("message", this.handleMessagesBound);
 
+        // the output frame is getting the code from the server and it knows
+        // whether it's good to run or not
         $el.find("#output-frame").on("load", function() {
             this.outputState = "clean";
-            this.markDirty();
+            //this.markDirty();
```

Comment:
This line can be removed.  This initial ```markDirty()``` is no longer required because output.html starts running code independently of index.html.  It's only when the user starts making changes to index.html that ```markDirty()``` comes into play.